docs/modules/ROOT/pages/advanced-networking.adoc

```diff
@@ -28,4 +28,6 @@ The following example shows an example configuration for configuring WAN sockets
 include::ROOT:example$/advanced-networking-wan.yaml[]
 ----
 
-Possible values for `serviceType` are: `ClusterIP`, `NodePort`, `LoadBalancer` and `ExternalName`. After applying this example configuration, there will be 1 service created with type of `NodePort` which exposes 5 ports: `5710`, `5711`, `5712`, `5713`, `5714`.
\ No newline at end of file
+Possible values for `serviceType` are: `ClusterIP` and `LoadBalancer`. After applying this example configuration, there will be 1 service created with type of `NodePort` which exposes 5 ports: `5710`, `5711`, `5712`, `5713`, `5714`. If you don't provide a `serviceType`, it will be set to `LoadBalancer` by default.
```

Comment:
```suggestion
Possible values for `serviceType` are: `ClusterIP` and `LoadBalancer`. After applying this example configuration, there will be 1 service created with a configured type that exposes 5 ports: `5710`, `5711`, `5712`, `5713`, `5714`. If you don't provide a `serviceType`, it will be set to `LoadBalancer` by default.
```
NodePort is not supported, AFAIK.

Reply:
Replaced it with `LoadBalancer`